workflows-scripts/circulation/check-in-check-out/scripts/circ-data-load.sh

```diff
@@ -0,0 +1,59 @@
+# This script needs to be supplied a psql.conf file, which contains the endpoint and credentials of the database.
+# This script needs to be run in the same directory as the loans.tsv, requests.tsv, and patron_action_sessions.tsv"
+# Example usage: $ ./circ-data-load.sh psql.conf fs08000010 
+
+#!/bin/bash -e
+
+if [[ $# -lt 2 ]]; then
+   echo "Usage: $0 <db config> <TENANT> [--clear-loan-audit]";
+   echo "       $0 psql.conf diku";
+   
+   exit 1;
+fi
+
+DBCONF=${1}
+export TENANT=${2}
+
+export PGDATABASE=`cat $DBCONF | jq '.database' | cut -d\" -f2`
+export PGUSER=`cat $DBCONF | jq '.username' | cut -d\" -f2`
+export PGPASSWORD=`cat $DBCONF | jq '.password' | cut -d\" -f2`
+export PGHOST=`cat $DBCONF | jq '.host' | cut -d\" -f2`
+export PGPORT=`cat $DBCONF | jq '.port' | cut -d\" -f2`
+
+LOANS="loans.tsv"
+REQUESTS="requests.tsv"
+# AUDIT_LOANS="audit_loans.tsv"
+PATRON_ACTION_SESSION="patron_action_sessions.tsv"
+
+echo "================================================="
+echo "PGUSER: $PGUSER"
+echo "PGPASSWORD: $PGPASSWORD"
+echo "PGDATABASE: $PGDATABASE"
+echo "PGHOST: $PGHOST"
+echo "PGPORT: $PGPORT"
+echo "TENANT: $TENANT"
+echo ""
+echo "LOANS: $LOANS"
+echo "REQUESTS: $REQUESTS"
+echo "AUDIT_LOANS: $AUDIT_LOANS"
+echo "PATRON_ACTION_SESSION: $PATRON_ACTION_SESSION"
+echo "=================================================="
+
+# ==================================================
+#  re-populate the database
+# ==================================================
+
+test "null" != "${LOANS}" && psql -a -c "\copy ${TENANT}_mod_circulation_storage.loan(id, jsonb) FROM '${LOANS}' DELIMITER E'\t'"
+test "null" != "${REQUESTS}" && psql -a -c "\copy ${TENANT}_mod_circulation_storage.request(id, jsonb) FROM '${REQUESTS}' DELIMITER E'\t'"
+# test "null" != "${AUDIT_LOANS}" && psql -a -c "\copy ${TENANT}_mod_circulation_storage.audit_loan(id, jsonb) FROM '${AUDIT_LOANS}' DELIMITER E'\t'"
+test "null" != "${PATRON_ACTION_SESSION}" && psql -a -c "\copy ${TENANT}_mod_circulation_storage.patron_action_session(id, jsonb) FROM '${PATRON_ACTION_SESSION}' DELIMITER E'\t'"
+
+# Update the Items and set their status to "checked out"

```

Comment:
Why do you update status to "check out" if this script is used in check out workflow also?